### PR TITLE
feat(message): support multiple channel IDs

### LINKED ISSUE #118: Support multiple channel IDs
### Description

The `channel-id` input supports a single channel ID. Can this be updated to support multiple comma-separated channel ID's?

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [ ] bug
- [x] enhancement (feature request)
- [ ] question
- [ ] documentation related
- [ ] example code related
- [ ] testing related
- [ ] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### PR DESCRIPTION
**Pull request by treemmett**
###  Summary

Adds support for comma-separated channel ID's.

Closes #118 

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,22 @@ Using JSON payload for constructing a message is also available:
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+Multiple channels can be sent the same message by providing a comma-separated list of ID's:
```
Comment (treemmett) [inline README.md:127]: Any thoughts on how to reword this?

File: src/slack-send.js [hunk 2]
```diff
@@ -60,25 +60,27 @@ module.exports = async function slackSend(core) {
...
 
       if (message.length > 0 || payload) {
         const ts = core.getInput('update-ts');
-        if (ts) {
+        await Promise.all(channelIds.split(',').map(async (channelId) => {
+          if (ts) {
           // update message
-          webResponse = await web.chat.update({ ts, channel: channelId, text: message, ...(payload || {}) });
-        } else {
+            webResponse = await web.chat.update({ ts, channel: channelId.trim(), text: message, ...(payload || {}) });
```
Comment (filmaj) [inline src/slack-send.js:76]: Nice, good call to trim it here.

File: test/slack-send-test.js [hunk 3]
```diff
@@ -56,16 +56,19 @@ describe('slack-send', () => {
     describe('happy path', () => {
       it('should send a message using the postMessage API', async () => {
         fakeCore.getInput.withArgs('slack-message').returns('who let the dogs out?');
-        fakeCore.getInput.withArgs('channel-id').returns('C123456');
+        fakeCore.getInput.withArgs('channel-id').returns('C123456,C987654');
```
Comment (filmaj) [inline test/slack-send-test.js:59]: Thanks for adding the test! Could we actually split this test case up into two test cases? Ideally I'd like separate tests for the single-channel and multi-channel scenarios.

**Review by filmaj — CHANGES_REQUESTED** [review]
Thanks a lot for the PR @treemmett !

I left a couple of comments, one suggestion on how to integrate the new feature you are adding support for in the README and another one regarding test organization.

Going to approve the github action execution - feel free to at-mention me directly if you'd like for me to take another look!

File: README.md (hunk 1 above)
Comment (mwbrooks) [reply README.md:127]: Yep, good suggestion @filmaj! ➕

**Review by mwbrooks — COMMENTED** [review]
Beauty, thanks for the contribution @treemmett!

**Comment by treemmett** [comment]
@filmaj Thanks for the suggestions! They've been implemented and pushed.

**Review by filmaj — APPROVED** [review]
Awesome! Thanks a ton for putting this together. Gonna let the tests run then will merge in!

**Comment by amirkarimi** [comment]
@treemmett Thanks for implementing this feature. We're trying to update the messages sent to multiple channels and we get `Error: Error: An API error occurred: message_not_found`. It updates the message in one channel but not in the other one.

**Pull request merged**